lib/api/src/grpc/proto/points.proto

```diff
@@ -41,8 +41,13 @@ message PointId {
   }
 }
 
+message ValuesIndices {
+  repeated uint32 data = 1;
+}
+
 message Vector {
   repeated float data = 1;
+  optional ValuesIndices indices = 2;
```

Comment:
will this encoding allow empty sparse vectors?
empty values & empty indices/

Reply:
It seems empty sparse vector is avaliable. For empty sparse vector `ValuesIndices` must be presented and `ValuesIndices.data` must be empty